tasks/release.py

```diff
@@ -720,6 +720,9 @@ def update_release_json(new_version: Version, max_version: Version):
 
     release_entry = release_entry_for(new_version.major)
     print(f"Updating {release_entry} for {new_version}")
+    # If the release isn't a RC, update the last stable release field
+    if not new_version.is_rc():
```

Comment:
Should it be part of the `_update_release_json` method instead?

Reply:
Indeed that seems more sensible, I"ll update